sklearnex/_utils.py

```diff
@@ -113,3 +114,20 @@ def get_hyperparameters(self, op):
         return estimator_class
 
     return wrap_class
+
+
+# This abstract class is meant to generate a clickable doc link for classses
+# in sklearnex that are not part of base scikit-learn. It should be inherited
+# before inheriting from a scikit-learn estimator, otherwise will get overriden
+# by the estimator's original.
+class IntelEstimator(ABC):
+    @property
+    def _doc_link_module(self) -> str:
+        return "sklearnex"
+
+    @property
+    def _doc_link_template(self) -> str:
+        module_path, _ = ".".join(self.__class__.__module__.rsplit(".", 1))

```

Comment:
```suggestion
        module_path, _ = self.__class__.__module__.rsplit(".", 1)

```
Sorry, made a mistake in my original suggestion. Totally my fault.